src/olympia/addons/tests/test_theme_update.py

```diff
@@ -36,10 +36,13 @@ def setUp(self):
     def test_wsgi_application_200(self, LWThemeUpdate_mock,
                                   MigratedUpdate_mock):
         MigratedUpdate_mock.return_value.is_migrated = False
+        LWThemeUpdate_mock.return_value.get_json.return_value = u'{"fo": "bá"}'
         # From AMO we consume the ID as the `addon_id`.
         for path_info, call_args in six.iteritems(self.urls):
             environ = dict(self.environ, PATH_INFO=path_info)
-            theme_update.application(environ, self.start_response)
+            response = theme_update.application(environ, self.start_response)
+            # gunicorn chokes on a unicode string response.
```

Comment:
Don't mention `gunicorn` as we shouldn't be using that. We should definitely return a bytes string here, that's what is expected by WSGI.

Reply:
https://github.com/mozilla/addons-server/pull/10960/commits/8417d9fc2a04aea886aba3bec34676ddcb433430